### ISSUE TITLE
Entries not deleted when file was deleted

### ISSUE THREAD

**Issue by mouradvt04** [rayyanqcri/rayyan-store#72]
Ticket 4681:
"Hi. I accidentally uploaded an excel file with all of my references plus about 1,000,000 lines that had a number in the "key" column but no other references. I deleted that file from the "Search methods" but those references are still included in "Inclusion decisions". Is there any way to delete all of the references that I uploaded from that spreadsheet?"
Checking the corresponding relation r63120:
postgres> select file_ids, count(*) from r63120 group by  file_ids;
+----------------------+---------+
| file_ids             | count   |
|----------------------+---------|
| ["242388", "242402"] | 2422    |
| ["219032"]           | 3798    |
| ["242388"]           | 1046153 |
| ["242402"]           | 8909    |
+----------------------+---------+

the deleted file (242388) is still there and it could be because it has exact matches with (242402) or something else. I asked the user to create a new review and keep this one so we can debug.

**Comment by hammady** [comment]
1M is too large. Probably the deletion query times out after deleting some and keeps on repeating until it reaches the max attempts of 5.

**Comment by mouradvt04** [comment]
Yes, it is, see https://rayyan.qcri.org/admin/delayed_jobs/159553486. 
It wasn't able to delete any.
Shouldn't we be more robust to this kind of situations?

**Comment by hammady** [comment]
We could have made it more complex and robust but such situations are very rare to justify the complexity. We need to think of a creative solution here.

**Comment by mouradvt04** [comment]
Agree.
One more thing is that we shouldn't consume empty entries in the first time.
